students/nartowska_karina/lesson_02_flow_control/factorial.py

```diff
@@ -0,0 +1,6 @@
+value = int(input())
+i = 1
```

Comment:
There is no need to set an initial value for "i" variable in a case when it is used as an iterator in the "for" loop.